### PR TITLE
Update minikube.rst

### PR DESCRIPTION
**Pull request by ackerman80**
Replace the Cluster IP by the service name in the HTTP GSG

**How to test (optional)**:
Follow instructions in the GSG

Signed-off-by: Arvind Soni <arvindsoni@gmail.com>

### CODE REVIEW

File: Documentation/gettingstarted/minikube.rst [hunk 1]
```diff
@@ -187,9 +187,9 @@ From the perspective of *deathstar* service providers only the ships with label
 
 .. parsed-literal::
 
-    $ kubectl exec xwing-cc65988f5-7cvn8 -- curl -s -XPOST 10.109.254.198/v1/request-landing
+    $ kubectl exec xwing-cc65988f5-7cvn8 -- curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
```
Comment (tgraf) [inline Documentation/gettingstarted/minikube.rst:190]: Does just referring to it as `deathstar` not work?

**Review by tgraf — CHANGES_REQUESTED** [review]

**Comment by ackerman80** [comment]
test-docs-please

File: Documentation/gettingstarted/minikube.rst (hunk 1 above)
Comment (ackerman80) [reply Documentation/gettingstarted/minikube.rst:190]: no it isn't working in the minikube. I checked the previous demo app and that was referring to the service the same way.

**Review by tgraf — APPROVED** [review]

**Comment by ackerman80** [comment]
test-me-please

**Pull request merged**